### DISCUSSION TITLE
univer可以在vue2种使用吗？文件是文件流时怎样转换成data进行渲染，可以帮忙解答一下吗

### DISCUSSION THREAD

**Discussion by minbaowang** [dream-num/univer#2213] [Q&A | 常见问题] [answered]
univer可以在vue2种使用吗？
   项目是vue2项目，但是要使用这个显示excel，看文档都是ts，js写法有吗？

文件是文件流时怎样转换成data进行渲染。
  之前用luckysheet时是通过LuckyExcel.transformExcelToLucky()，来进行数据转换的，但是数据量大时，这个方法转换的时间较长，请问univer有新的渲染方法吗？

谢谢解答！

**Comment by jikkai** [comment]
官网有提供 Vue 2 的 demo

https://univer.ai/playground/?title=Vue%202

**Reply by minbaowang** [reply]
文件流怎样转成所需data，在哪里可以看到吗？

**Reply by Aarona-hub** [reply]
解决了吗哥们

**Reply by zhang-ke-ke** [reply]
解决了吗？不搭建他们的后端服务，求他们把导入导出从后端服务里面剥离出来，只留协同编辑需要部署就行了

**Reply by shiguoqian** [reply]
同问，求在前端能直接转换的方式

**Reply by zhang-ke-ke** [reply]
没有找到，他们的后端服务也只支持1MB，多了就要买商业化的了，不过他们的前身[luckysheet]可以，但是只支持一个实例，创建多个就不行了

**Comment by hexf00** [comment, marked as answer]
How to Open or Edit an Excel File from a URL

https://univer.ai/guides/sheet/features/import-export#how-to-open-or-edit-an-excel-file-from-a-url

**Reply by lanfaboy** [reply]
Why would you report it?:Cannot find "univer-pro.exchange-client.exchange.service" registered by any injector. The stack of dependencies is: "univer-pro.exchange-client.exchange.service".

**Reply by zhang-ke-ke** [reply]
因为需要搭建他们提供的部署平台
